src/OpenTelemetry.Api.ProviderBuilderExtensions/README.md

```diff
@@ -6,7 +6,7 @@
 ## Installation
 
 ```shell
-dotnet add package --prerelease OpenTelemetry.Api.ProviderBuilderExtensions
+dotnet add package OpenTelemetry.Api.ProviderBuilderExtensions
```

Comment:
lets keep this separate. The changelog points to this PR, so someone following it should be shown only whats really changed. (These are just noise for them)